public/app/plugins/panel/geomap/style/markers.ts

```diff
@@ -95,6 +111,7 @@ const makers: SymbolMaker[] = [
           radius,
           angle: Math.PI / 4,
         }),
+        text: !cfg?.text ? undefined : textLabel(cfg),
```

Comment:
```suggestion
        text: textLabel(cfg),
```
and have the function return undefined when missing